doc/releases/changelog-0.36.0.md

```diff
@@ -11,53 +11,80 @@
   [(#5154)](https://github.com/PennyLaneAI/pennylane/pull/5154)
   [(#5464)](https://github.com/PennyLaneAI/pennylane/pull/5464)
   [(#5465)](https://github.com/PennyLaneAI/pennylane/pull/5465)
+  [(#5278)](https://github.com/PennyLaneAI/pennylane/pull/5278)
+  [(#5384)](https://github.com/PennyLaneAI/pennylane/pull/5384)
 
-  Focusing on the [spectral norm](https://en.wikipedia.org/wiki/Matrix_norm)
-  error, it is possible to specify errors in the following operations:
+  Two new user-facing classes are now available to allow for calculating and propagating 
+  gate errors in PennyLane:
 
-  * For algorithmic-level usecases it is useful to create a custom operation representing a
-    major building block of the algorithm. This building block can have an error attached but
-    does not require a decomposition or matrix representation:
+  * `qml.resource.error.SpectralNormError`: the spectral norm error is defined as the 
```

Comment:
Please can we always go with the minimal import depth. This can be accessed through `qml.resource.SpetralNormError`

Reply:
```suggestion
  * `qml.resource.SpectralNormError`: the spectral norm error is defined as the 
```

Reply:
Note the same also for `ErrorOperation` and the `from pennylane.resource.error` import (though need to check)